infra/mgr_provision.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Create meshdb_mgmt directory
+MGMT_DIR="/opt/meshdb_mgmt"
+mkdir -p $MGMT_DIR
+cd $MGMT_DIR
+
+# Clone the repo
+apt-get update && apt-get install -y git unzip
+git clone https://github.com/nycmeshnet/meshdb.git
+
+# Install tf
+wget https://releases.hashicorp.com/terraform/1.8.2/terraform_1.8.2_linux_amd64.zip
+unzip terraform_*
+mv terraform /usr/bin/
+
+# JBO TODO REMOVE DEBUG
+cd meshdb
+git checkout james/infra_updates
+cd ..
+# END DEBUG
+
+# Setup secret files (will need to be modified)
+cp meshdb/infra/helm/meshdb/secret.values.yaml ./secret.values.yaml
+cp meshdb/infra/helm/meshdb/values.yaml ./values.yaml
+cp meshdb/infra/tf/example.tfvars ./local.tfvars
```

Comment:
You might want to call this out in the README as well

Reply:
This one doesn't need to exist (the vm already exists)
The other ones are mentioned as `../../../../values.yaml` and `../../../../secret.values.yaml`